testing/suite/integration.go

```diff
@@ -262,4 +272,8 @@ func dumpResourceInfo(t *testing.T, namespace string) {
 	descrCSC := NewKubectlCommandLine("describe", "clustersupplychains")
 	descrCSC.Exec()
 	t.Logf("Describe clustersupplychains \n %s", descrCSC.GetOutput())
+	// describe pod
+	descrPod := NewKubectlCommandLine("describe", "pod", "pod1", "-n", TestingNamespace)
+	descrPod.Exec()
+	t.Logf("Describe clustersupplychains \n %s", descrCSC.GetOutput())
```

Comment:
this line is printing the previous command output.

Reply:
also update the command to `kubectl describe pods -n TestingNamespace`

Reply:
Done